install/scripts/install-llvm.sh

```diff
@@ -1,11 +1,32 @@
+#!/bin/bash 
+HIPSYCL_PKG_LLVM_REPO_BRANCH=${HIPSYCL_PKG_LLVM_REPO_BRANCH:-release/9.x}
 export INSTALL_PREFIX=${INSTALL_PREFIX:-/opt/hipSYCL}
 
 set -e
 BUILD_DIR=$HOME/git/llvm-vanilla
 rm -rf $BUILD_DIR
 
-#git clone -b release/9.x https://github.com/llvm/llvm-project $BUILD_DIR
-git clone -b release/10.x https://github.com/llvm/llvm-project $BUILD_DIR
+
+case $HIPSYCL_PKG_LLVM_REPO_BRANCH in
+
+	release/9.x)
+		echo "Cloning LLVM $HIPSYCL_PKG_LLVM_REPO_BRANCH"	
+		git clone -b $HIPSYCL_PKG_LLVM_REPO_BRANCH https://github.com/llvm/llvm-project $BUILD_DIR
+		sed -i 's/CHECK_SIZE_AND_OFFSET(ipc_perm, mode);//g' $BUILD_DIR/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc
```

Comment:
This is only necessary for clang 9? Wouldn't it be easier to just clone in any case, and then, if we are on clang 9, additionally apply the patch? This would simplify the code compared to a switch statement and we wouldn't have to modify the code if we want to transition to a new clang version.

Reply:
You're right, I'll change that